docs/5.0/architecture/authentication.md

```diff
@@ -72,11 +72,11 @@ certificate on a node makes it a cluster member.
    dynamic, single-use token generated by [`tctl nodes
    add`](../cli-docs.md#tctl-nodes-add).
 
-    !!! tip "Token TTL"
+!!! tip "Token TTL"
 
-        When using dynamic tokens, their default time to live (TTL) is 15
-        minutes, but it can be reduced (not increased) via
-        [`tctl nodes add --ttl`](../cli-docs.md#tctl-nodes-add) flag.
+    When using dynamic tokens, their default time to live (TTL) is 15
```

Comment:
The default actually appears to be 30 minutes rather than 15: https://github.com/gravitational/teleport/blob/6a82501dc615119146a552ca2a98d76fda5fbc7d/lib/defaults/defaults.go#L148-L151

```suggestion
    When using dynamic tokens, their default time to live (TTL) is 30
```